crates/cadenza-syntax/src/generated/snapshots/cadenza_syntax__generated__test_data__invalid_parse__record_missing_brace_comma__invalid_parse_record_missing_brace_comma_cst.snap

```diff
@@ -0,0 +1,29 @@
+---
+source: crates/cadenza-syntax/src/generated/test_data.rs
+expression: "{ a = 1,\nfoo\n"
+---
+Root@0..13
+  Apply@0..9
+    LBrace@0..1 "{"
+    Space@1..2 " "
+    ApplyReceiver@2..2
+      SyntheticRecord@2..2
+    ApplyArgument@2..7
+      Apply@2..7
+        ApplyArgument@2..4
+          Identifier@2..3
+            Identifier@2..3 "a"
+          Space@3..4 " "
+        ApplyReceiver@4..5
+          Equal@4..5
+            Equal@4..5 "="
+        Space@5..6 " "
+        ApplyArgument@6..7
+          Literal@6..7
+            Integer@6..7
+              Integer@6..7 "1"
+    Comma@7..8 ","
+    Newline@8..9 "\n"
+  Identifier@9..12
```

Comment:
I would expect there to be an Error node inserted right here saying that we expected to see a `}`